token/invocation/options.go

```diff
@@ -44,6 +44,15 @@ func WithMeta(key string, val any) Option {
 	}
 }
 
+// WithEncryptedMeta adds a key/value pair in the "meta" field.
+// The value is encrypted with the given aesKey.
+// Accepted types for the value are: string, []byte.
+func WithEncryptedMeta(key string, val any, encryptionKey []byte) Option {
```

Comment:
If you prefer that, yes. Personally, I'd like just one method, since in `AddEncrypted` I'm doing a type assertion anyway, to make sure it's either string or []byte.

Reply:
Done ✔️